models/game_captains.py

```diff
@@ -8,15 +8,8 @@
 from models.game import Game
 from models.no_player_action_exception import NoPlayerActionException
 
-
-embed_template = discord.Embed(
-    title='Captains will now pick teams!',
-    colour=discord.Colour.teal()
-)
-embed_template.set_footer(
-    text='UEA Private Matches by curpha',
-    icon_url='https://cdn.akamai.steamstatic.com/steamcommunity/public/images/avatars/be/bed810f8bebd7be235b8f7176e3870de1006a6e5_full.jpg'
-)
+from embed_temp import embed_temp
+embed_template = embed_temp.Embed_Template(embed_title='Captains will now pick teams!').emb_temp
```

Comment:
Please can module imports be at the top of the file.
If all the previous suggestions are done, you will be able to do something like this:
```
import embed_template

...

embed = embed_template.get_template()
```

Reply:
Hey great to hear the feedback, I'll make the changes right away. Just saying I am sort of a beginner currently but I'm continuously learning.

Reply:
No worries! After you've done this issue, if you want to start a new one let me know so I can give some tips on how it should be done :)

Reply:
Actually I have a question on the ```@staticmethod``` decorator and how I should implement that in the EmbedTemplate class.

Reply:
Sure what is your question?

Reply:
![image](https://user-images.githubusercontent.com/87343367/142941258-741f7dc6-f9a5-4511-bb36-8c8113952c33.png)

I'm thinking about doing it this way but let me know what you think.

Reply:
So to make sure, does that work for what you wanted?

Reply:
Well, I would think that the footer would indeed have to be set each time that you would want the message to have the footer.

Reply:
And I can set both the title and color as arguments required in order to initialize the EmbedTemplate class.

Reply:
Can you take another look at the final commits? I think this issue is about taken care of but let me know.

Reply:
If you override the discord.EmbedClass, classing super and then setting the footer, you would then use this class instead of discords embed meaning you would only have to set the footer once. 

What you have done is good and would work! But I am of the opinion of something that works can be done better it should be

Reply:
Yeah, personally I would agree with that statement as well. Also, could you explain what you mean by overriding and classing super?

Reply:
Sure! All you have to do is pass in the discord.Embed class into the class declaration like this: 
class EmbedTemplate(discord.Embed)

Then call the discord.Embed super function inside \_\_init\_\_:
self.super(EmbedTemplate, self).\_\_init\_\_()

Reply:
I think that’s the syntax for it anyway, if not I’m sure it isn’t far off lol

Reply:
I didn't encounter any errors so I'm assuming that's the right syntax

Reply:
Perfect, then replace all references to discord.Embed with EmbedTemplate. Not sure if you’ll have to put embed_template.EmbedTemplate or not, you’ll have to see what works

Reply:
Do you mean references to discord.Embed inside the class itself?

Reply:
Ok, so to make sure, wouldn't that be making the EmbedTemplate class into a subclass of the discord.Embed class?

Reply:
If the goal is indeed to make the EmbedTemplate a subclass of discord.Embed, then it seems from what I've found that the syntax would be a little different and the title and colour can be made attributes of self